### PR TITLE
feat(website): render `@defaultValue` blocks

### PR DESCRIPTION
**Pull request by suneettipirneni**
**Please describe the changes this PR makes and why it should be merged:**

Members with `@defaultValue` tags will show their default value in the docs.

### CODE REVIEW

**Comment by Syjalo** [comment]
Shouldn't it show something like `Default value: {value}`?

**Comment by suneettipirneni** [comment]
> Shouldn't it show something like `Default value: {value}`?

It does:
Screen Shot 2022-08-19 at 12 29 44 PM

I assume you're viewing the PR preview, which didn't have the updated api.json being used.

**Comment by Syjalo** [comment]
> > Shouldn't it show something like `Default value: {value}`?
> 
> It does: Screen Shot 2022-08-19 at 12 29 44 PM
> 
> I assume you're viewing the PR preview, which didn't have the updated api.json being used.

Does this also apply to local launch?

**Comment by suneettipirneni** [comment]
> Does this also apply to local launch?

Yes, if you build the relevant packages before running the website locally it should use the updated api.json.

**Comment by Syjalo** [comment]
Double quotes (\`\`Node.js $process.version\`\`)
image

File: packages/rest/src/lib/REST.ts [hunk 1]
```diff
@@ -25,45 +25,45 @@ export interface RESTOptions {
...
 	 *
-	 * @default 'Bot'
+	 * @defaultValue `'Bot'`
 	 */
 	authPrefix: 'Bot' | 'Bearer';
 	/**
 	 * The cdn path
 	 *
-	 * @default 'https://cdn.discordapp.com'
+	 * @defaultValue 'https://cdn.discordapp.com'
```
Comment (Syjalo) [inline packages/rest/src/lib/REST.ts:41]: You forgot to put the quotes

**Pull request merged**